floyd/model/experiment.py

```diff
@@ -68,6 +68,9 @@ def localize_date(self, date):
             date = utc.localize(date)
         return date.astimezone(PST_TIMEZONE)
 
+    def to_dict(self):
```

Comment:
this method name is a little bit misleading since it's returning a str representation of json not a python dictionary object.